docs/cow-protocol/reference/core/auctions/ebbo_specifics.md

```diff
@@ -0,0 +1,49 @@
+---
+id: ebbo-rules
+---
+
+# EBBO violations: details
+
+In this section, we will elaborate on the details about what consistutes an EBBO violation, and what actions are taken by the DAO in case such a violation occurs. The content of this section is based on [CIP-52](https://snapshot.org/#/cow.eth/proposal/0x0f2f1fde68d85081a7d60f7ac99dafbdabdbf8c8cf55961f2609b3dff429a24a).
+
+
+## Certificate of EBBO violation
+
+A certificate for an EBBO violation consists of a reference routing on a block (and log index) between the start of the auction and when the settlement happened onchain. A reference routing for a trade at a given block (and index) is an execution of that trade which only uses liquidity from base protocols and routes through base tokens (see [here](/cow-protocol/reference/core/auctions/competition-rules#base-tokens) for the definition of base protocols and base tokens). The surplus received by users in this routing is used as reference surplus. The difference between the reference surplus and the surplus actually received by the user is the size of the EBBO violation. This amount needs to be reimbursed to a user.
+
+A certificate for an EBBO violation can be challenged by the solver who is accused of the EBBO violation by providing a different block (and index), within 72 hours of the notification of the violation. In this case, a reference routing on this block (and index) might be proposed by the core team and used as certificate instead. The new certificate, if any, cannot be challenged again.
```

Comment:
I find the "in this case" part confusing. In the case of a solver challenging the core team is also proposing another certificate?

Why not allow it to be challenged by anyone?

Also small advice, if you make every sentence it's own line it doesn't change formatting but will make changes later easier to review (since `diff` doesn't work well on a single line).

Reply:
This was the phrasing in the CIP, and the intention was to avoid a lot of back and forth. I think anyone can challenge certificates, the core team should just verify the claim, and this can basically be done once. Which suggests that the solver that is "accused" of an EBBO violation should provide a definitive proof that there was no violation; if that proof can be challenged again, we conclude that there was an EBBO violation.

Not sure if that answers your concern.